src/commandsHandler.ts

```diff
@@ -13,13 +14,14 @@ export const SAVE_SNIPPET_COMMAND = "Tabnine.saveSnippet";
 export async function registerCommands(
   context: ExtensionContext
 ): Promise<void> {
-  context.subscriptions.push(
-    commands.registerCommand(CONFIG_COMMAND, openHub(StateType.PALLETTE))
-  );
-
-  context.subscriptions.push(
-    commands.registerCommand(STATUS_BAR_COMMAND, handleStatusBar(context))
-  );
+  if (!ONPREM) {
```

Comment:
remove it from here and just do not call `registerCommands` from the extensionOnPrem.ts

Reply:
some of the commands need to be registered

Reply:
what commands? enableSaveSnippetContext is not releavant